### PR TITLE
8265407: JFR: Fix Typos

### PR DESCRIPTION
**Pull request by egahlin**
Hi,

Could I have a review of a fix that takes care of couple of typos.

Testing: test/jdk/jdk/jfr/

Thanks
Erik

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8265407](https://bugs.openjdk.java.net/browse/JDK-8265407): JFR: Fix Typos

### Reviewers
 * [Markus Grönlund](https://openjdk.java.net/census#mgronlun) (@mgronlun - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3562/head:pull/3562` \
`$ git checkout pull/3562`

Update a local copy of the PR: \
`$ git checkout pull/3562` \
`$ git pull https://git.openjdk.java.net/jdk pull/3562/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3562`

View PR using the GUI difftool: \
`$ git pr show -t 3562`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3562.diff">https://git.openjdk.java.net/jdk/pull/3562.diff</a>

</details>

### CODE REVIEW

File: src/jdk.management.jfr/share/classes/jdk/management/jfr/RemoteRecordingStream.java [hunk 1]
```diff
@@ -83,7 +82,7 @@
  *         rs.onEvent("jdk.GCPhasePause", System.out::println);
  *         rs.start();
  *     }
- * }
+ *     }
```
Comment (mgronlun) [inline src/jdk.management.jfr/share/classes/jdk/management/jfr/RemoteRecordingStream.java:85]: Indentation?

**Review by mgronlun — APPROVED** [review]
Looks good.

File: src/jdk.management.jfr/share/classes/jdk/management/jfr/RemoteRecordingStream.java (hunk 1 above)
Comment (egahlin) [reply src/jdk.management.jfr/share/classes/jdk/management/jfr/RemoteRecordingStream.java:85]: I want it to align with the starting brace.

**Pull request closed without merging**